modules/light-clients/07-tendermint/light_client_module_test.go

```diff
@@ -247,8 +260,9 @@ func (suite *TendermintTestSuite) TestVerifyUpgradeAndUpdateState() {
 			if expPass {
 				suite.Require().NoError(err)
 
-				clientStateBz := clienttypes.MustMarshalClientState(suite.chainA.Codec, upgradedClientState)
-				suite.Require().Equal(upgradedClientStateBz, clientStateBz)
+				clientStateAny, err := codectypes.NewAnyWithValue(upgradedClientState)
+				suite.Require().NoError(err)
+				suite.Require().Equal(upgradedClientStateAny.Value, clientStateAny.Value)
```

Comment:
These checks are kinda silly, and were before. I'm going to change to read the clientstate from store again and then compare bz

Reply:
https://github.com/cosmos/ibc-go/pull/5967/commits/1298b8935521555899b672a484772b378906620c